src/controllers/Auth.controller.ts

```diff
@@ -114,7 +156,48 @@ export const resetPasswordRequestController = async (
       .status(404)
       .json({ status: 404, success: false, message: "Email does not exist" });
   } else {
-    const resetToken = await generateToken(user.dataValues.email, "5m");
+    const {
+      id,
+      twoFAenabled,
+      twoFAverified,
+      secret,
+      surname,
+      email,
+      role,
+      status,
+      confirmationCode,
+      googleId,
+      province,
+      district,
+      sector,
+      cell,
+      street,
+      avatar,
+      lastPasswordUpdate,
+    } = user;
+
+    const resetToken = await generateToken(
+      {
+        id,
+        twoFAenabled,
+        twoFAverified,
+        secret,
+        surname,
+        email,
+        role,
+        status,
+        confirmationCode,
+        googleId,
+        province,
+        district,
+        sector,
+        cell,
+        street,
+        avatar,
+        lastPasswordUpdate,
+      },
+      "5m",
```

Comment:
@ISHIMWE-Aime 

There is a concern for this.

A short expiration date for tokens is more secure. But there is a catch.

We don't have `refresh-tokens` set in place. So, this means that a user will be forced to log in every 5 minutes. This is a terrible experience for the user.

I would suggest to increase this time, because we don't have refresh tokens set up, and we are out of time to implement that.